code/modules/cargo/packs.dm

```diff
@@ -1143,63 +1148,169 @@
 	desc = "For when you REALLY have to reinforce something."
-	cost = 16500
+	cost = 17400
 	contains = list(/obj/item/stack/sheet/plasteel/fifty)
 	crate_name = "plasteel sheets crate"
 
+/datum/supply_pack/materials/copper20
+	name = "20 Copper Sheets"
+	desc = "Makes your floors look nice and your circuitry run!"
+	cost = 800
+	contains = list(/obj/item/stack/sheet/mineral/copper/twenty)
+	crate_name = "copper sheets crate"
+
+/datum/supply_pack/materials/copper50
+	name = "50 Copper Sheets"
+	desc = "Makes your floors look nice and your circuitry run!"
+	cost = 2400
+	contains = list(/obj/item/stack/sheet/mineral/copper/fifty)
+	crate_name = "bulk copper sheets crate"
+
+/datum/supply_pack/materials/silver20
+	name = "20 Silver Sheets"
+	desc = "Somewhat less shiny."
+	cost = 1700
+	contains = list(/obj/item/stack/sheet/mineral/silver/twenty)
+	crate_name = "silver sheets crate"
+
+/datum/supply_pack/materials/silver50
+	name = "50 Silver Sheets"
+	desc = "Somewhat less shiny."
+	cost = 3650
+	contains = list(/obj/item/stack/sheet/mineral/silver/fifty)
+	crate_name = "bulk silver sheets crate"
+
+/datum/supply_pack/materials/gold20
+	name = "20 Gold Sheets"
+	desc = "Shiny!"
+	cost = 3200
+	contains = list(/obj/item/stack/sheet/mineral/gold/twenty)
+	crate_name = "gold sheets crate"
+
+/datum/supply_pack/materials/gold50
+	name = "50 Gold Sheets"
+	desc = "Shiny!"
+	cost = 7400
+	contains = list(/obj/item/stack/sheet/mineral/gold/fifty)
+	crate_name = "bulk gold sheets crate"
+
+/datum/supply_pack/materials/uranium20
+	name = "20 Uranium Sheets"
+	desc = "Green rock make grug puke red!"
+	cost = 2700
+	contains = list(/obj/item/stack/sheet/mineral/uranium/twenty)
+	crate_name = "uranium sheets crate"
+
+/datum/supply_pack/materials/uranium50
+	name = "50 Uranium Sheets"
+	desc = "Green rock make grug puke red!"
+	cost = 6150
+	contains = list(/obj/item/stack/sheet/mineral/uranium/fifty)
+	crate_name = "bulk uranium sheets crate"
+
+/datum/supply_pack/materials/titanium20
+	name = "20 Titanium Sheets"
+	desc = "Useful for big stompy mechs and advanced tools!"
+	cost = 3200
+	contains = list(/obj/item/stack/sheet/mineral/titanium/twenty)
+	crate_name = "titanium sheets crate"
+
+/datum/supply_pack/materials/titanium50
+	name = "50 Titanium Sheets"
+	desc = "Useful for big stompy mechs and advanced tools!"
+	cost = 7400
+	contains = list(/obj/item/stack/sheet/mineral/titanium/fifty)
+	crate_name = "titanium sheets crate"
+
+/datum/supply_pack/materials/diamond5
+	name = "5 Diamond Sheets"
+	desc = "If you like high technology, these materials can help!"
+	cost = 3200
+	contains = list(/obj/item/stack/sheet/mineral/diamond/five)
+	crate_name = "diamond sheets crate"
+
+/datum/supply_pack/materials/diamond20
+	name = "20 Diamond Sheets"
+	desc = "If you like high technology, these materials can help!"
+	cost = 11600
+	contains = list(/obj/item/stack/sheet/mineral/diamond/twenty)
+	crate_name = "bulk diamond sheets crate"
+
```

Comment:
Stuff like this shouldn't be buyable and should remain the job of the miners

Reply:
Titanium is eh but diamonds and uranium probably not so much, the station is in orbit around a planet rich in materials for a reason so nanotrasen shouldn't be shipping materials to them but the mining outpost going to nanotrasen

Reply:
I think otherwise but I will atomize. All those packs have now been removed.